resources/views/dashboard/training/new_ticket.blade.php

```diff
@@ -19,124 +19,28 @@
             <div class="col-sm-3">
                 <div class="form-group">
                     {!! Form::label('controller', 'Controller', ['class' => 'form-label']) !!}
-                    @if($c != null)
-                        {!! Form::select('controller', $controllers, $c, ['placeholder' => 'Select Controller', 'class' => 'form-control']) !!}
-                    @else
-                        {!! Form::select('controller', $controllers, null, ['placeholder' => 'Select Controller', 'class' => 'form-control']) !!}
+                    @if (!is_null($c))
+                        $c = null;
```

Comment:
This is causing `$c = null;` to display - so probably not doing what it's supposed to here

<img width="332" alt="image" src="https://github.com/ZTL-ARTCC/Current_Website/assets/38896380/e6722bf2-bc6a-46be-a679-1b7e2756ebf2">

Reply:
What was I thinking... this must have been some late-night coding. Anyway, fixed. This just makes sure that if the value is nullish that it is nullified.